sagemaker-python-sdk/scikit_learn_model_registry_batch_transform/scikit_learn_model_registry_batch_transform.ipynb

```diff
@@ -0,0 +1,671 @@
+{
```

Comment:
<p>Is any cleanup needed to delete models, anything in Model Registry, etc? I believe there's no endpoints/ongoing resources so I don't think the cleanup is essential, but please double-check to ensure customers won't incur ongoing charges.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3086/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
No cleanup needed. Customers are not billed for Model Registry.